admin_manual/maintenance/migrating.rst

```diff
@@ -62,141 +62,138 @@ When a user tries a URL that is not whitelisted the following error appears:
-
+   #
+   #     Edit ssh-config (enable root ssh login)
+   #
    nano /etc/ssh/sshd_config
-
-Change PermitRootLogin to "yes"::
-
+   #
+   #     Change PermitRootLogin to "yes"
+   #
    PermitRootLogin yes
-   
-Restart ssh service::
-
+   #
+   #     Restart ssh service
+   #
    service ssh stop
+   #
    service ssh start
-
-Install ownCloud on new server
+   #
+   #     Install ownCloud on new server
 
 =========
 Migration
 =========
 
-1. Put original server in maintenance mode:
-
-Go in owncloud dir::
+1. Put original server in maintenance mode::
 
+      #     Go in owncloud dir
+      #
       cd /var/www/owncloud/
-
-Switch to maintenance mode::
-
+      #
+      #     Switch to maintenance mode
+      #
       sudo -u www-data php occ maintenance:mode --on
+      #
+      #     wait for 6-7 min and stop apache2
+      #
+      service apache2 stop
 
-wait for 6-7 min and stop apache2::
-
-   service apache2 stop
-
-2. Transfer the database
-
-Go in owncloud dir::
+2. Transfer the database::
 
+      #     Go in owncloud dir
+      #
       cd /var/www/owncloud/
-
-Backup the database::
-
-   mysqldump --single-transaction -h localhost -u admin -ppassword owncloud > owncloud-dbbackup.bak
+      #
+      #     Backup the database
+      #
+      mysqldump --single-transaction -h localhost -u admin -ppassword owncloud > owncloud-dbbackup.bak
+      #
+      #     Export the database **to** new server
+      #
+      rsync -Aaxt owncloud-dbbackup.bak root@new_server_address:/var/www/owncloud 
+      #
+      #     Import the database **on** new server
+      #
+      mysql -h localhost -u admin -ppassword owncloud < owncloud-dbbackup.bak
 
 .. note:: You can find the values for the mysqldump command in your config.php at your owncloud directory.
 [server]= dbhost, [username]= dbuser, [password]= dbpassword, and [db_name]= dbname.
 
-.. note:: For InnoDB tables only:
-The --single-transaction flag will start a transaction before running. Rather than lock the entire database, this will let mysqldump read the database in the current state at the time of the transaction, making for a consistent data dump.
+.. note:: For InnoDB tables only: The --single-transaction flag will start a transaction before running. 
+Rather than lock the entire database, this will let mysqldump read the database in the current state at the time of the transaction, making for a consistent data dump.
 
 For Mixed MyISAM / InnoDB tables:
-Either dumping your MyISAM tables separately from InnoDB tables or use --lock-tables instead of --single-transaction to guarantee the database is in a consistent state when using mysqldump.
-
-
-Export the database **to** new server::
-
-   rsync -Aaxt owncloud-dbbackup.bak root@new_server_address:/var/www/owncloud 
-
-Import the database **on** new server::
-
-   mysql -h localhost -u admin -ppassword owncloud < owncloud-dbbackup.bak
+Either dumping your MyISAM tables separately from InnoDB tables or use --lock-tables
```

Comment:
would be good to quote the switch, i.e., ``--lock-tables`` (and --single-transaction as well).